docs/about.md

```diff
@@ -1,3 +1,5 @@
+!!! info This page contains guidance to implementers and is not part of the normative-track [FHIRcast specification](../specification).
```

Comment:
Is that supposed to be some special formatting (!!! info)? It doesn't seem like it gets applied when I try and view the page

Reply:
Thanks, Will. Fixed. Yes, it should look like this:
![image](https://user-images.githubusercontent.com/60514/89321251-4e6d5980-d648-11ea-9225-bf4ccad34843.png)